tests/fixture-data/consim/genesis.json

```diff
@@ -1,328 +1 @@
-      "debug_static_validators": false,
-      "reward_factor_epoch_election_any": "0"
-    }
-  },
-  "beacon": {
-    "params": {
-      "debug_deterministic": false
-    }
-  },
-  "consensus": {
-    "backend": "tendermint",
-    "params": {
-      "timeout_commit": 1000000000,
-      "skip_timeout_commit": false,
-      "empty_block_interval": 0,
-      "max_tx_size": 32768,
-      "max_block_size": 22020096,
-      "max_block_gas": 0,
-      "max_evidence_age": 100000
-    }
-  },
-  "halt_epoch": 18446744073709551615,
-  "extra_data": null
-}
+{"height":0,"genesis_time":"2020-07-16T12:21:45.321530685Z","chain_id":"test","epochtime":{"params":{"interval":86400,"debug_mock_backend":false},"base":0},"registry":{"params":{"gas_costs":{"deregister_entity":1000,"register_entity":1000,"register_node":1000,"register_runtime":1000,"runtime_epoch_maintenance":1000,"unfreeze_node":1000,"update_keymanager":1000},"max_node_expiration":5},"entities":[{"account":{"v":0,"signers":[{"public_key":"TqUyj5Q+9vZtqu10yw6Zw7HEX3Ywe0JQA9vHyzY47TU=","weight":1}],"threshold":1},"signatures":["ZkMI7Pbn/ZsdLzH1F/d1mMz1XY4O1KWU/AyYRvgbJ8O90FLuYyvV+CvxakhCcnzwXg+bTgxdZt59hxu50Z1lBw=="],"payload":"o2F2AW9hY2NvdW50X2FkZHJlc3NVAKn/rA3/YzKBP4g3oB1YQ063Z7w3eBlhbGxvd19lbnRpdHlfc2lnbmVkX25vZGVz9Q=="}]},"roothash":{"params":{"gas_costs":{"compute_commit":1000,"merge_commit":1000}}},"staking":{"params":{"thresholds":{"entity":"0","node-compute":"0","node-keymanager":"0","node-storage":"0","node-validator":"0","runtime-compute":"0","runtime-keymanager":"0"},"commission_schedule_rules":{},"min_delegation":"0","fee_split_weight_propose":"0","fee_split_weight_vote":"1","fee_split_weight_next_propose":"0","reward_factor_epoch_signed":"0","reward_factor_block_proposed":"0"},"token_symbol":"TEST","token_value_exponent":0,"total_supply":"200000000000","common_pool":"0","last_block_fees":"0","ledger":{"oasis1qz5lltqdla3n9qfl3qm6q82cgd8tweauxucvwlzp":{"general":{"balance":"100000000000"},"escrow":{"active":{"balance":"100000000000","total_shares":"1"},"debonding":{"balance":"0","total_shares":"0"},"commission_schedule":{},"stake_accumulator":{}}}},"delegations":{"oasis1qz5lltqdla3n9qfl3qm6q82cgd8tweauxucvwlzp":{"oasis1qz5lltqdla3n9qfl3qm6q82cgd8tweauxucvwlzp":{"shares":"1"}}}},"keymanager":{},"scheduler":{"params":{"min_validators":1,"max_validators":100,"max_validators_per_entity":1,"debug_bypass_stake":false,"debug_static_validators":false,"reward_factor_epoch_election_any":"0"}},"beacon":{"params":{"debug_deterministic":false}},"consensus":{"backend":"tendermint","params":{"timeout_commit":1000000000,"skip_timeout_commit":false,"empty_block_interval":0,"max_tx_size":32768,"max_block_size":22020096,"max_block_gas":0,"max_evidence_age_blocks":100000,"max_evidence_age_time":172800000000000,"state_checkpoint_interval":10000,"state_checkpoint_num_kept":2,"state_checkpoint_chunk_size":8388608,"gas_costs":{"tx_byte":1}}},"halt_epoch":18446744073709551615,"extra_data":null}
```

Comment:
why did we have so many accounts before?

Reply:
Don't remember, it doesn't need them.